crux-bench/src/crux/bench/watdiv_datomic.clj

```diff
@@ -354,4 +354,7 @@
                         (->> (run-watdiv-bench {:test-count 100})
                              (filter :query-idx)
                              (sort-by :query-idx)))
-    (bench/save-to-s3 {:database "datomic" :version "0.9.5697"} output-file)))
+    (bench/save-to-s3 {:database "datomic" :version "0.9.5697"} output-file))
+  (shutdown-agents)
+  ;; TODO: Queries with timeouts don't close properly (currently), so this system exit is necessary. Post a fix, it should be removed.
```

Comment:
is this still the case with the `release` -> `finally` change above?

Reply:
No, I don't imagine it would be - The issue seems to be primarily with queries, if I were to remove the querying the program exits just fine.